charts/app-config/templates/kubescape.yaml

```diff
@@ -13,13 +13,18 @@ spec:
     repoURL: 'https://github.com/kubescape/helm-charts'
     targetRevision: HEAD
     helm:
-      values: |
+      valueFiles:
+        - values.yaml
```

Comment:
Is values.yaml not included by default?

Reply:
following this example https://github.com/kubescape/helm-charts/blob/b54d494a7631214f4355636fb36b551e908c611f/ArgoCDApplication.yaml

Reply:
Saying that the values.yaml should be included by default